docs/book/common_code/images/activity_machineset_reconciliation.drawio

```diff
@@ -0,0 +1 @@
+<mxfile modified="2019-03-20T22:50:36.347Z" host="www.draw.io" agent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.121 Safari/537.36" etag="2AuruRjm7a1qkO0jtuUg" version="10.5.0" type="google"><diagram id="7pWM9kC39Dyr3UhoOzVi" name="Page-1">7Vxbd5s4EP41fkyPkLj5sbm0PWeb7W7ysM0jBmFoMfKC3Nj761cywoBkYmzA2DR5aNEgAR7NfPPNMPYE3S3WnxNnGTwSD0cTCLz1BN1PINS0qc3+45KNkABbzyTzJPSErBA8h//hfKKQrkIPp5WJlJCIhsuq0CVxjF1akTlJQl6r03wSVe+6dOZYETy7TqRK/wk9GmRS2wCF/AsO50F+Zw2IMwsnnywEaeB45LUkQg8TdJcQQrOjxfoOR1x7uV6ydZ9qzu4eLMExbbLA+LF+mX0PXM//07n9O3l8evm6vhFX+eVEK/GBxcPSTa4BdhWmbDa4DegiYjKNHbIPs+TnU+ok9Jk6lJ/3wyi6IxFJtgsR2P7xyTQhP3HpjO+LM+LuOKF4XfuxtJ2ymJlhssA02bAp+QJgZEuEhZli+FrsFhI7EJQ2Kpc5wj7muwsXKmQHQotHaBQ20KjHTEwMSUIDMiexEz0U0rKmuXJCZo8fo3AeM9mMUEoW7ASOvY/cwPlFljjOJMKB7De1zjSbbL4zIfhg5MMXPswH9+vKaCNG2QfhT1/Zq5SsEhcfNjJlT5krO8kc0zeWov17n+DIoeGv6nPs20qx9C8SsjvvbMYGFZPRTMkYsscSiwp7YOp2NqVpSz4hrb+NBqX7QLvqocfNZwfZExTGuVPJ6faKFHv9jOkWv9wgjHG6HZDZDw6usiEnZBV72BOW+hqEFD8vna0tvLJ4UDVkJ3GFdercmHwS031gIYGIz/7c3kHElBSvww8qjOy2p4wjel84or/jSI2pnoIjRi84IjssLOzmAJR05b2GYiWRM9tysBRHzGXZnm2piBtwEvVpAs2Iqel2xsTmnO72ouzTAVnMVulhfx7EUS1J4Zq1z1HBHke1+nJUU9mCmLRzVUe4aIR92rPnahW/Ldz4OM/d45EHndlo6LkZ7tUaRE4uWnqyKVMCoPXDCUwgWbCB3uQE8nwNoDOQAkux6Q1OezVqSpaXFIv6tGi7kUFvqvyidaSqXhZBq2GkUi51U823bqDkKMT3t6Sx+2BnK1Y537JTN1qlFCfb9D6RqOtvwlgRHJqvTpXNaYcXI+Crdi2WHMQIDezf/Y4Jqw7PS1fzoFqyksJ58TpM+bMyauosuCtm/9YN41majQF5jXHyhP3yWkFuefQcF+XVcx0ORng1tWh4RYwXTIZivPnGHXb+mmy1W8q7CwW5YelnxgK1UvpOMrszIjQdhGZWbcqQy+yXTzI1taQiaqB7okxNhALZillE3J/f+LJ7HGEakni8fFRBk8EJqaYWZsbOSE8ClKZpax2gtIQMJBU4jIaU9NjCi3IfAEQG3PR1zAkrIKyu6Kdao6mJMcOAjFSLXJgd3X79dvfH7wNAhg4Gf4ejjTgpZtkaLS1joxfxkPy4WMQHrV8gt8ikYT+ZNJIKMIaJTi2oNbhUpqHuMXAKDyDawRX9IBpU6wTpJnaZ5AkvI+YBvx+emWh4PINq5j8aPDsVm/INOAmb+mlvkZ3WKkLhmXL73C5O7ivjuz1kV5luVBNQDcB9LSHn7SxTW3VWS48rSerWSZnmVmolZbTQaMELgMar6tfppAZ6GBpbNOxAsxdo1KUMzZwaDaGxnmt1Dp5q4SJduS5mrnvt7zJk9VvAGLx/B6rNDlf0OuPIBp48a8vztNKqY7O2U4pNuWlfSNePZUu943ZTNDjIuWxN4VwdJXFyR5Cla28+2TAdRFCtSY317U6HlZB+fWqIN0KmVNywjKa5yAW9E4JqHc8nfFe4ekRhwvGzgutjGD9hx9s8Y5fEXtZ0e10xWkEypEZofU+EZjlSTyEaqeWg9xDdF5xMLypE26hqjbqOGqHHsVHVkHJK23y7MCpH1WPnnycKI7VgN9YoPIQLIOsCIqptSZe4/HiK1JQrwf+uMBuMtmi169sT6p4aw3/HDKkkvR02jKCen9vmSUWrmpDYdQxEzWJgZ+6q0t/ju3YBZ2zA441QnDfTcIFTyk9eFzuWunGNfW1OZy1f6VfNjYfrxkVNg7zW7Ps6bbtx5dYh89zv7PTr44qdm8OZqZzcL2aipt8fviAyp0PFbhzPK2O9h9m2enj7uH7IrIXtLQ8dIe+ljTHm/G60vM+Ut3j4d5W6+iL5gmlfJxHioP/nVnxBtE9GBws1DQmtf6gEIKtdmxcbFr9slE0vfiAKPfwP</diagram></mxfile>
```

Comment:
This appears to show that the MachineSet is only sync'd if a cluster does not exist or there is a DeletionTimestamp.

Reply:
I retitled the decision to be "no linked cluster" instead of "cluster exist"